carma-messenger-ui/website/widgets/truckInspection/widget.js

```diff
@@ -313,6 +318,22 @@ CarmaJS.WidgetFramework.truckInspection = (function () {
 
             console.log("truckInspectionSafetyLogPerVIN element is created");
     }
+
+    var IsValidVINNumber = function(vin_number)
+    {
+        let isValid = false;
+        console.log("Check VIN: " + vin_number);
+        let valid_truck_vin_numbers = CarmaJS.Config.getValidTruckVINNumber();
+        valid_truck_vin_numbers.forEach((valid_vin)=>{
+            console.log("Valid VIN: " + valid_vin);
```

Comment:
This is not valid VIN yet. So don't think you need this extra log. Or just don't say it's Valid yet.

Reply:
Addressed